TombEngine/Objects/TR2/Vehicles/skidoo.cpp

```diff
@@ -402,13 +403,13 @@ namespace TEN::Entities::Vehicles
 		else
 			laraItem->Pose.Orientation.x = laraItem->Pose.Orientation.z = 0;
 
-		AnimateItem(laraItem);
+		AnimateItem(*laraItem);
 
 		if (!dead && drive >= 0 && banditSkidoo)
 			SkidooGuns(skidooItem, laraItem);
 
 		if (!dead)
-			SyncVehicleAnimation(*skidooItem, *laraItem);
+			SyncVehicleAnim(*skidooItem, *laraItem);
```

Comment:
I still don't fully understand the logic of renaming `Animation` to `Anim`. We have a function named `SetAnimation`, and I think it is okay. Can we use full words in function names, please?